components/container-cards/category-card.scss

```diff
@@ -0,0 +1,109 @@
+//Delete  me
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
+
+.msds-container-card-category {
+  font-family: 'Open Sans';
+  cursor: pointer;
+  position: relative;
+  width: 280px;
+
+  &:hover {
+    .msds-container-card-category__header {
+      color: $yellow;
+    }
+  }
+
+  &:active {
+    .msds-container-card-category__header {
+      box-shadow: 0 0 0 4px rgba(255, 255, 255, 0.5);
```

Comment:
Should this shadow come from a variable?

Reply:
We dont have a variable setup for rgba(255, 255, 255, 0.5)